Source/Blazorise/Components/Validation/Validation.razor.cs

```diff
@@ -286,15 +286,15 @@ protected virtual Type DetermineHandlerType()
 
                 if ( Validator is not null || AsyncValidator is not null )
                 {
-                    return typeof( ValidatorValidationHandler );
```

Comment:
I'm not sure if this is the type of stuff that nowadays the compiler is smart enough to "cache" it... Because I mean... there isn't much it could do differently at runtime is there? Making it so this implementation is redudant? Can probably take this for a spin on the benchmark project.

Reply:
I guess I had the wrong idea, there's a benefit to it it seems, altough this is the that type of stuff that's quite negligible haha 
Still an improvement eitherway.
![image](https://user-images.githubusercontent.com/22283161/187558228-75134076-6850-4f68-963e-0f395c747b94.png)